cmsdb/campaigns/run2_2016_HIPM_nano_uhh_v12/ewk.py

```diff
@@ -89,9 +89,9 @@
 )
 
 cpn.add_dataset(
-    name="dy_lep_pt100To250_amcatnlo",
```

Comment:
I think it is a good idea to have all dataset names as all lower case.

However, changing the name of processes will impact all ongoing analyses because users will need to rename all of their datasets + all outputs that were produced using this dataset. Therefore, I have two comments to add:

- How can we implement such a change without impacting ongoing analyses too much? Can we provide some script that automatically changes the path of all outputs that have been produced with one of the affected datasets?
- When implementing this convention, we should require this for all datasets. Are these all datasets with uppercase letters or are there still datasets/processes remaining with uppercase letters? Maybe we should even add this requirement to our tests?

Reply:
Not yet. I added an aux flag `"allow_uppercase_name": True` to the ggHH processes that bypasses the lower case test. Since these are mostly bbww processes and datasets, I'd leave this to you @mafrahm. So feel free to rename them at some point and remove the flags again.

Reply:
Btw, I noticed a lot of inconsistencies in the names of processes / datasets in cases of open intervals on, for instance, `pt` or `mgg`. When there is just a single number in the dataset, `pT30`, this usually translates into `pt30toinf`. We should talk about this in the next meeting.